### PR TITLE
Add ChatGPT plugin

### PR DESCRIPTION
**Pull request by MichielvanBeers**
I've created a plugin that allows the user to make use of the ChatGPT gpt-3.5 and gpt-4 API's from FlowLauncher.

Looking forward to your review!

### CODE REVIEW

**Comment by Garulf** [comment]
Looks like the test failed: 

```

2023-03-18T23:35:05.7292609Z ##[group]Run python ./ci/src/test-python.py
2023-03-18T23:35:05.7293048Z [36;1mpython ./ci/src/test-python.py[0m
2023-03-18T23:35:05.7332159Z shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
2023-03-18T23:35:05.7332507Z env:
2023-03-18T23:35:05.7332790Z   pythonLocation: C:\hostedtoolcache\windows\Python\3.8.10\x64
2023-03-18T23:35:05.7333142Z ##[endgroup]
2023-03-18T23:35:06.3162294Z Traceback (most recent call last):
2023-03-18T23:35:06.3170480Z   File "./ci/src/test-python.py", line 167, in <module>
2023-03-18T23:35:06.3244728Z     manifest = plugin_reader()
2023-03-18T23:35:06.3245551Z   File "D:\a\Flow.Launcher.PluginsManifest\Flow.Launcher.PluginsManifest\ci\src\_utils.py", line 44, in plugin_reader
2023-03-18T23:35:06.3246134Z     return json.load(f)
2023-03-18T23:35:06.3246695Z   File "C:\hostedtoolcache\windows\Python\3.8.10\x64\lib\json\__init__.py", line 293, in load
2023-03-18T23:35:06.3247198Z     return loads(fp.read(),
2023-03-18T23:35:06.3248073Z   File "C:\hostedtoolcache\windows\Python\3.8.10\x64\lib\json\__init__.py", line 357, in loads
2023-03-18T23:35:06.3248714Z     return _default_decoder.decode(s)
2023-03-18T23:35:06.3249257Z   File "C:\hostedtoolcache\windows\Python\3.8.10\x64\lib\json\decoder.py", line 337, in decode
2023-03-18T23:35:06.3249797Z     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
2023-03-18T23:35:06.3250376Z   File "C:\hostedtoolcache\windows\Python\3.8.10\x64\lib\json\decoder.py", line 353, in raw_decode
2023-03-18T23:35:06.3250890Z     obj, end = self.scan_once(s, idx)
2023-03-18T23:35:06.3251429Z json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1384 column 5 (char 73829)
2023-03-18T23:35:06.5473889Z ##[error]Process completed with exit code 1.

```

File: plugins.json [hunk 1]
```diff
@@ -1369,5 +1369,17 @@
...
+        "ID": "df432fe0-9ff7-4ba1-b0fd-5ffd26efbf86",
+        "Name": "ChatGPT",
+        "Description": "Plugin to use OpenAI's ChatGPT in Flow Launcher",
+        "Author": "MichielvanBeers",
+        "Version": "1.0.0",
+        "Language": "python",
+        "Website": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT",
+        "IcoPath": "https://raw.githubusercontent.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT/main/Images/app.png",
+        "UrlSourceCode": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT",
+        "UrlDownload": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT/releases/download/v1.0.0/Flow.Launcher.Plugin.HelloWorldPython.zip",
```
Comment (Garulf) [inline plugins.json:1383]:
```suggestion
        "UrlDownload": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT/releases/download/v1.0.0/Flow.Launcher.Plugin.HelloWorldPython.zip"
```
Remove extra `,`.

**Review by Garulf — APPROVED** [review]

File: plugins.json [hunk 2]
```diff
@@ -1369,5 +1369,17 @@
...
+        "ID": "df432fe0-9ff7-4ba1-b0fd-5ffd26efbf86",
+        "Name": "ChatGPT",
+        "Description": "Plugin to use OpenAI's ChatGPT in Flow Launcher",
+        "Author": "MichielvanBeers",
+        "Version": "1.0.0",
+        "Language": "python",
+        "Website": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT",
+        "IcoPath": "https://raw.githubusercontent.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT/main/Images/app.png",
+        "UrlSourceCode": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT",
+        "UrlDownload": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT/releases/download/v1.0.0/Flow.Launcher.Plugin.HelloWorldPython.zip"
```
Comment (jjw24) [inline plugins.json:1383]: Hey @MichielvanBeers, please change the artifact name, once you published a new release you will also need to correct it here.

File: plugins.json [hunk 3]
```diff
@@ -1369,5 +1369,17 @@
...
+    },
+    {
+        "ID": "df432fe0-9ff7-4ba1-b0fd-5ffd26efbf86",
+        "Name": "ChatGPT",
+        "Description": "Plugin to use OpenAI's ChatGPT in Flow Launcher",
+        "Author": "MichielvanBeers",
+        "Version": "1.0.0",
+        "Language": "python",
+        "Website": "https://github.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT",
+        "IcoPath": "https://raw.githubusercontent.com/MichielvanBeers/Flow.Launcher.Plugin.ChatGPT/main/Images/app.png",
```
Comment (jjw24) [inline plugins.json:1381]: Also please use the cdn link, see another plugin's IcoPath as an example :)

File: plugins.json (hunk 2 above)
Comment (MichielvanBeers) [reply plugins.json:1383]: Oops, how sloppy, thanks @jjw24! I've created a new PR.

File: plugins.json (hunk 3 above)
Comment (MichielvanBeers) [reply plugins.json:1381]: Same here. I've created a new PR that set the IcoPath to make use of jsdelivery. Thanks!

**Pull request merged**